scripts/itkdev-docker-compose-completion.bash

```diff
@@ -1,6 +1,9 @@
 #/usr/bin/env bash
 _idc_completions()
 {
+  local cur
+  _get_comp_words_by_ref -n : cur
```

Comment:
When trying this, I get the error `_get_comp_words_by_ref: command not found`. Are there any requirements that must be fulfilled before using this command?

Reply:
It's an new part of bash-completion. Found this online:

> Installing bash-completion (1.2) through brew (brew install bash-completion) solved the issue for me, it might work for you too.

Reply:
If that fixes the issue we should add that as an notice to the readme file.